demo/client/src/camera.ts

```diff
@@ -306,6 +311,10 @@ export class PerspectiveCamera extends Camera {
         ]);
     }
 
+    zoom(scale: number): void {
```

Comment:
Why this change?

Reply:
I wanted to keep the old `zoom` function private, but I needed something public that only took the `scale` argument. So I renamed the private function to `_zoom` and exposed the new function as `zoom`.